docs/governance/templates/release_issue.md

```diff
@@ -6,7 +6,7 @@ and copy it into a release issue. Fill in relevent values, found inside {}
 {version} should be replaced with the version ie: 0.5.0.
```

Comment:
I think this file can be skipped.  I'm turning it into a issue template in #280

Reply:
Removed!